_includes/meta.html

```diff
@@ -24,15 +24,21 @@
   {% assign page_title = title | append: ' | 18F ' %}
 {% endif %}
 
+{% comment %}
+  Descriptions are escaped because some contain HTML content which,
+  left to its own devices, renders in a way that messes up <head>
+  and looks quite bad.
+{% endcomment %}
```

Comment:
Oh boy. Do we know _why_ some descriptions have HTML content? This sounds like an anti pattern to me

Reply:
It sounds like an anti-pattern to me too! And no, I don't understand why.

This is one of those things that's outside the scope of the migration, that is, to try to understand why the content is the way it is with the intent to change it. I've commented where there are "hotspots" of concerning practices or code smells, but the migration would ideally result in an identical product to the current production 18F site.

Reply:
Totally fair. SGTM